src/ubuntu/22.04/cross/illumos/Dockerfile

```diff
@@ -1,5 +1,7 @@
 ARG ROOTFS_DIR=/crossrootfs/x64
 
+# dummy comment to trigger the build
+
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local AS builder
```

Comment:
This is intentional to get newest changes in. We updated it in arcade.

Reply:
This one https://github.com/dotnet/arcade/pull/14978.

Reply:
Ok, that's fine. Can we change it to be just a whitespace change rather than leaving that comment in there?

Reply:
Sure, though I'm not sure if it will trigger the build with something like this:

```suggestion
FROM  mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local AS builder
```

Reply:
It will. Any commit change to the file will trigger.